phpcs.xml.dist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<ruleset name="PHP AI Client Coding Standards">
+    <description>PSR-12 coding standards for PHP AI Client SDK</description>
+
+    <!-- Scan these files -->
+    <file>src</file>
+
+    <!-- Ignore vendor directory -->
+    <exclude-pattern>*/vendor/*</exclude-pattern>
+
+    <!-- Use PSR-12 standard -->
+    <rule ref="PSR12"/>
```

Comment:
As a note, here, we can encourage folks to use [PER-3.0](https://www.php-fig.org/per/coding-style/) in the CONTRIBUTING doc, as that extends PSR-12, but unfortunately [PHPCS doesn't support PER](https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/29).